### PR TITLE
Feature/text area

### PR DESCRIPTION
**Pull request by geemanjs**

### CODE REVIEW

File: src/forms/__tests__/textarea.spec.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { render } from 'enzyme'
+import TextArea from 'govuk-react-components/forms/textarea';
+
+describe('The Input Text Component', () => {
```
Comment (lennym) [inline src/forms/__tests__/textarea.spec.js]: This refers to the wrong component

File: src/forms/__tests__/textarea.spec.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    ).toMatchSnapshot();
+  });
+
+  it('should render an error when an error is passed in', () => {
+    expect(
+      render(<TextArea label="ip" name="ip" value="someText" error="Testing error" />)
+    ).toMatchSnapshot();
+  });
+
+  it('should render as a checked input correctly', () => {
```
Comment (lennym) [inline src/forms/__tests__/textarea.spec.js]: I'm not sure this makes sense. What is "a checked input" in the context of a textarea?

Comment (lennym) [reply src/forms/__tests__/textarea.spec.js]: Looking further into the code, "checked" appears to be "react speak" for "has an `onChange`. 

In which case this test is insufficient to test that, as the presence of an `onChange` prop doesn't affect the html output at all. The test will need to be more sophisticated than a simple html snapshot.

As an aside, I'd try to stay away from tests that are just described by "renders correctly" as it doesn't provide any actual indication of the intent.

Comment (geemanjs) [reply src/forms/__tests__/textarea.spec.js]: Corrected :+1: Take a look now :)

**Pull request merged**